projects/elife.yaml

```diff
@@ -597,20 +535,9 @@ journal:
                 origins:
                     CloudFrontCDNOrigin:
                         hostname: prod--journal.elifesciences.org
-                    LogInOrigin:
-                        hostname: prod--journal.elifesciences.org
-                        pattern: /log-in
-                        headers:
-                            - X-Requested-With
-                            - Host
-                            - Referer
-                        cookies:
-                            - journal
-                logging:
-                    bucket: elife-cloudfront-logs
             fastly:
                 subdomains:
-                    - "{instance}--fastly-journal"
+                    - "{instance}--cdn-journal"
                     - "placeholder-prod-journal" # only here to avoid rewriting the following values
```

Comment:
Unrelated, but not sure what this comment refers to.

Reply:
The DNS entries are numbered in the CloudFormation template:
```
$ cat .cfn/stacks/journal--prod.json | jq '.Resources | keys' | grep DNS
  "CloudFrontCDNDNS1",
  "CloudFrontCDNDNS2",
  "CloudFrontCDNDNS3",
  "CloudFrontCDNDNS4",
  "ExtDNS",
  "FastlyDNS1",
  "FastlyDNS2",
  "FastlyDNS3",
```
So if `placeholder-prod-journal` were to be removed, the following DNS entry would have to be deleted and re-added, probably in two-steps, causing unnecessary trouble.

Reply:
Do we create that value or CloudFormation?

Reply:
Would using a hash rather than an auto-increment have avoided the problem? (I get we can't change it now easily.)

Reply:
Is reordering a safe operation? If so, could it be reordered so the placeholder is last, then remove it as a follow up?

Reply:
Re-ordering not safe as every resource is independent, so re-additions would likely clash with removals done at the same time - no clear solution at the moment, generating something better than an incremental number as a function of the value would make it more stable but complicated to port to that